ProcessMaker/Repositories/CaseParticipatedRepository.php

```diff
@@ -103,4 +91,13 @@ public function updateStatus(int $caseNumber, array $statusData)
             \Log::error($e->getMessage());
         }
     }
+
+    private function checkIfCaseParticipatedExist(int $userId, int $caseNumber): bool
+    {
+        $this->caseParticipated = CaseParticipated::where('user_id', $userId)
+            ->where('case_number', $caseNumber)
+            ->first();
```

Comment:
Use `exists()` instead of `first` since it does not require to get the row to verify if the record exists.

Reply:
the `$this->caseParticipated` property is used to update the JSON fields

Reply:
gotcha